openedx/core/djangoapps/credit/partition_schemes.py

```diff
@@ -0,0 +1,141 @@
+"""
+Provides partition support to the user service.
+"""
+
+import logging
+
+from course_modes.models import CourseMode
+from lms.djangoapps.verify_student.models import SkippedReverification, VerificationStatus
+from student.models import CourseEnrollment
+
+
+log = logging.getLogger(__name__)
+
+
+class VerificationPartitionScheme(object):
+    """
+    This scheme assigns users into the partition 'VerificationPartitionScheme'
+    groups. Initially all the gated exams content will be hidden except the
+    ICRV blocks for a 'verified' student until that student skips or submits
+    verification for an ICRV then the related gated exam content for that ICRV
+    will be displayed.
+    Following scenarios can be handled:
+
+    verified_allow: When a student skips or submits an ICRV verification then that
```

Comment:
Please add groups docstrings in this order:
1. `non_verified`
2. `verified_allow`
3. `verified_deny`